### DISCUSSION TITLE
SDK 1.9.0 Privacy Attribute

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#850] [Q&A] [answered]
Hello,
What could be the effect of not implementing the “privacy” attribute in our application ?
How is this attribute supposed to be handled ? (compared to "mandatory" or "forbidden" for example)

Best regards

**Comment by YvesJo** [comment, marked as answer]
Hi,
I guess you are referring to the so-called "unpublish" fields. In their absence, it will not be possible to withheld some information and everything that will be submitted will get published.

**Reply by luispozoWS** [reply]
How does the "unpublished" fields work regarding the "privacy" attribute of a field ?
****Ex** : BT-88-Procedure**
Does the information declared by the buyer in the "unpublished fields" "feed" the privacy attribute of BT-88-Procedure ?
We would like to have more details regarding this issue to understand how to develop this
![image](https://github.com/OP-TED/eForms-SDK/assets/158502029/513d1d3a-8274-4456-a0e8-3ef6f29d0608)

Best regards

**Reply by YvesJo** [reply]
Hi,
There is no "privacy" attribute in the XML sense. The "privacy" section of fields.json identifies the "unpublish" fields that are associated to the current field "BT-88-Procedure". The code "pro-fea" correspond to the value unpublichedFieldId should have.
You may find some explanation and XML illustration in the documentation https://docs.ted.europa.eu/eforms/latest/schema/withheld-publication.html
